source/backend/dal/Repositories/ProjectRepository.cs

```diff
@@ -72,17 +72,17 @@ public Task<Paged<PimsProject>> GetPageAsync(ProjectFilter filter)
         /// <summary>
         /// Get by ID - Search Projects by Id.
         /// </summary>
-        /// <param name="id"></param>
+        /// <param name="projectId"></param>
         /// <returns></returns>
-        public async Task<PimsProject> Get(long id)
+        public async Task<PimsProject> Get(long projectId)
         {
-            User.ThrowIfNotAuthorized(Permissions.PropertyView);
+            User.ThrowIfNotAuthorized(Permissions.ProjectView);
 
             return await Context.PimsProjects
                     .AsNoTracking()
                     .Include(x => x.ProjectStatusTypeCodeNavigation)
                     .Include(x => x.RegionCodeNavigation)
-                    .Where(x => x.Id == id)
+                    .Where(x => x.Id == projectId)
                     .FirstOrDefaultAsync();
```

Comment:
Also not opposed to using **async** DB operations but I think we were not using that previously. Is performance better or is just dev preference to use async EF methods? Would like to sync up with other devs so we so the same

Reply:
thread should perform a little better but overall, not a big win. Async feels a little more modern approach, but we could keep not using it overall.

Reply:
Let's not do this. The request is spawned on its own thread already. So far all calls to this method are just waiting on it, making the async, not a worth having

Reply:
@eddherrera My PR will change this so no need to worry as we chatted earlier